mslib/msui/wms_control.py

```diff
@@ -1342,6 +1336,7 @@ def continue_retrieve_image(self, img, legend_img, layer, style, init_time, vali
         self.display_retrieved_image(img, legend_img, layer, style, init_time, valid_time, complete_level)
         # this is for cases where 'remove' button is clicked, then 'retrieve' is clicked
         self.signal_disable_cbs.emit()
+        self.image_displayed.emit()
```

Comment:
somehow I think we should keep the prefix now, and refactor (remove all prefix) if we think that is a good idea.

Now from reading I can match easier names which are based on emit() if they have this signal in its name.